cmd/loki/main.go

```diff
@@ -24,14 +24,14 @@ import (
 func main() {
 	var config loki.ConfigWrapper
 
+	if config.PrintVersion || len(os.Args[1:]) == 1 && os.Args[1] == "version" {
```

Comment:
```suggestion
	if len(os.Args) >= 1 && os.Args[1] == "-version" {
```

`config.PrintVersion` will never be set, and the arg should be `-version`